core/src/main/java/com/novoda/merlin/RxCallbacksManager.java

```diff
@@ -0,0 +1,25 @@
+package com.novoda.merlin;
+
+import rx.Observable;
+import rx.subjects.BehaviorSubject;
+
+public class RxCallbacksManager {
+
+    private final BehaviorSubject<Boolean> merlinRxConnectionStatusSubject;
```

Comment:
Maybe it could be worth having an enum with CONNECTED/DISCONNECTED ? Makes it more explicit than true/false and allow for extension if we need to handle transitional states in the future.